storage/src/store/operation.rs

```diff
@@ -136,6 +153,22 @@ impl<K: Array, V: Codec> Variable<K, V> {
     }
 }
 
+impl Write for Keyless {
+    fn write(&self, buf: &mut impl BufMut) {
+        match &self {
+            Keyless::Append(offset) => {
+                buf.put_u8(APPEND_CONTEXT);
```

Comment:
The preferred syntax here is `APPEND_CONTEXT.write` and `offset.write` (infers type).

Reply:
changed